index.html

```diff
@@ -99,7 +99,7 @@ <h5 class="card-h">KD Ignite</h5>
             <div class="section3-header">
                 <h3 class="section3-h">Featured Speakers</h3><br>
                 <hr class="break-line"/>
-            </div>
+            </div><br>
```

Comment:
**As I am not able to highlight the specific piece of code I will include it with sufficient photos**

- [ ] In your "Main Program" desktop version, kindly ensure that the individual cards have an on hover that has a highlight effect on the card when hovered over.

**This is the design your design currently is missing Notice card number 4:**

![image](https://user-images.githubusercontent.com/86318284/212086226-f33cf834-b751-4727-b472-23d51b44a3cf.png)

- [ ] On **Mobile view**, kindly add some padding/ margin top at the landing section to give it some spacing which will make it match the provided Behance design.

**This is how your mobile version landing page looks like:**

![image](https://user-images.githubusercontent.com/86318284/212087230-96bb5330-bca7-4ba0-a451-3716c0e38a0d.png)

**Note the difference with the Behance design:**

![image](https://user-images.githubusercontent.com/86318284/212087380-389ff568-e81f-4831-af2b-318166d1d795.png)


- [ ] Just as in **Mobile view landing section** mentioned above, your **Mobile view Featured Speakers** section requires some margin/ padding left and right to align them properly at the middle. This will make them match the provided Behance design.

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/212087890-09ddadcc-c1a9-48f9-82d9-47cbb2e47d04.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/212087980-56316bce-f2e4-4a6d-83d9-dd068f229d46.png)